src/components/Onfido/BaseOnfidoWeb.js

```diff
@@ -105,7 +105,7 @@ class Onfido extends React.Component {
                 Log.hmmm('Onfido user closed the modal');
             },
             language: {
-                locale: this.props.preferredLocale,
+                locale: this.props.preferredLocale === CONST.LOCALES.ES ? CONST.ONFIDO.ES_ES_LOCALE : CONST.LOCALES.EN,
```

Comment:
Expensify currently supports english and spanish only.
Please make it as below, so that other languages work if included in future

`locale: this.props.preferredLocale === CONST.LOCALES.ES ? CONST.ONFIDO.ES_ES_LOCALE : this.props.preferredLocale`

Reply:
@Prince-Mendiratta Is there any difference if we use `en` without a tag?